packages/ag-charts-community/src/chart/options/chartOptions.ts

```diff
@@ -103,6 +106,10 @@ export interface AgBaseThemeableChartOptions {
     highlight?: AgChartHighlightOptions;
     /** HTML overlays */
     overlays?: AgChartOverlaysOptions;
+    /** Configuration for the chart legend. */
+    legend?: AgChartLegendOptions;
+    animation?: AgAnimationOptions;
```

Comment:
Did you want to put comments for these?

Reply:
They didn't have comments originally, this is one for Laurence to follow up on I think.